numpy/core/src/common/simd/avx2/math.h

```diff
@@ -56,4 +56,23 @@ NPY_FINLINE npyv_f64 npyv_maxp_f64(npyv_f64 a, npyv_f64 b)
     return _mm256_blendv_pd(a, max, nn);
 }
 
+// Minimun, natively mapping with no guarantees to handle NaN.
```

Comment:
```suggestion
// Minimum, natively mapping with no guarantees to handle NaN.
```
Just a typo needs to get fixed across all the C comments in this patch.

Reply:
fixed!